### ISSUE TITLE
mpv build fails using brew on 10.10.2

### ISSUE THREAD

**Issue by twayz** [mpv-player/mpv#1497]
Hi,

I just got this error when trying to build mpv on yosemite 10.10.2. Also, it seems like other users are getting the same error and opened an issue on the mpv-homebrew repo (https://github.com/mpv-player/homebrew-mpv/issues/46). I'm not sure where I was supposed to submit the issue so I just wanted to let the devs know about it on here.

Thank you.

```
➜  ~  brew install --HEAD --with-official-libass --with-bluray-support --with-libdvdread --with-little-cms2 --with-lua mpv && brew linkapps
==> Installing mpv from mpv-player/homebrew-mpv
==> Cloning https://github.com/mpv-player/mpv.git
Updating /Library/Caches/Homebrew/mpv--git
==> Checking out branch master
==> Patching
==> Downloading https://pypi.python.org/packages/source/d/docutils/docutils-0.11.tar.gz
Already downloaded: /Library/Caches/Homebrew/mpv--docutils-0.11.tar.gz
tar: Failed to set default locale
==> python setup.py install --prefix=/usr/local/Cellar/mpv/HEAD/libexec
==> Downloading http://ftp.waf.io/pub/release/waf-1.8.4
Already downloaded: /Library/Caches/Homebrew/mpv--waf-1.8.4
==> python waf configure --prefix=/usr/local/Cellar/mpv/HEAD --enable-zsh-comp
==> python waf install
Waf: Leaving directory `/private/tmp/mpv-2lLNCV/build'
Build failed
 -> task in 'objects' failed (exit status 1):
	{task 4412858704: c gl_video.c -> gl_video.c.13.o}
['clang', '-D_ISOC99_SOURCE', '-D_GNU_SOURCE', '-D_LARGEFILE_SOURCE', '-D_FILE_OFFSET_BITS=64', '-D_LARGEFILE64_SOURCE', '-std=c99', '-Wall', '-O2', '-g', '-Werror=implicit-function-declaration', '-Wno-error=deprecated-declarations', '-Wno-error=unused-function', '-Wempty-body', '-Wdisabled-optimization', '-Wstrict-prototypes', '-Wno-format-zero-length', '-Werror=format-security', '-Wno-logical-op-parentheses', '-fcolor-diagnostics', '-Wno-tautological-compare', '-Wno-tautological-constant-out-of-range-compare', '-Wall', '-Wundef', '-Wmissing-prototypes', '-Wshadow', '-Wno-switch', '-Wno-parentheses', '-Wpointer-arith', '-Wredundant-decls', '-Wno-pointer-sign', '-pthread', '-I/private/tmp/mpv-2lLNCV/build', '-I/private/tmp/mpv-2lLNCV', '-I/usr/local/Cellar/libdvdread/5.0.0/include', '-I/usr/local/Cellar/little-cms2/2.6/include', '-I/usr/local/Cellar/ffmpeg/2.5.3/include', '-I/usr/local/Cellar/mpg123/1.21.0/include', '-I/usr/local/include', '-I/usr/local/Cellar/freetype/2.5.5/include/freetype2', '-I/usr/local/Cellar/fribidi/0.19.6/include/fribidi', '-I/usr/local/Cellar/fontconfig/2.11.1/include', '-I/usr/local/Cellar/freetype/2.5.3_1/include/freetype2', '-I/usr/local/Cellar/libass/0.12.1/include', '-DHAVE_DVDREAD=1', '-DHAVE_LCMS2=1', '-DHAVE_LIBSWRESAMPLE=1', '-DHAVE_LIBAV=1', '-DHAVE_LIBAVFILTER=1', '-DHAVE_MPG123=1', '-DHAVE_LIBAVDEVICE=1', '-DHAVE_52=1', '-DHAVE_LIBASS=1', '../video/out/gl_video.c', '-c', '-o', '/private/tmp/mpv-2lLNCV/build/video/out/gl_video.c.13.o']

READ THIS: http://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/homebrew):
  https://github.com/mpv-player/homebrew-mpv/issues
```

**Comment by Argon-** [comment]
Error:

    ../video/out/gl_video.c:389:32: error: use of undeclared identifier 'GL_RGB32F'
    ../video/out/gl_video.c:393:33: error: use of undeclared identifier 'GL_RGBA32F'

Replacing with `GL_RGB(A)32F_ARB` works.

**Comment by 191919** [comment]
Workaround.

```
diff --git a/video/out/gl_header_fixes.h b/video/out/gl_header_fixes.h
index d7c48b3..795da16 100644
--- a/video/out/gl_header_fixes.h
+++ b/video/out/gl_header_fixes.h
@@ -62,6 +62,8 @@
 #if defined __APPLE__ && !(defined GL_VERSION_3_0)
 #define GL_RGBA16F                        0x881A
 #define GL_RGB16F                         0x881B
+#define GL_RGBA32F                        0x8814
+#define GL_RGB32F                         0x8815
 #define GL_MAJOR_VERSION                  0x821B
 #define GL_MINOR_VERSION                  0x821C
 #define GL_NUM_EXTENSIONS                 0x821D
```

**Comment by pigoz** [comment]
fixed with 29cf62d201
